proposals/p1964.md

```diff
@@ -0,0 +1,282 @@
+Adding support for a specific character literal supports clean, readable,
+concises use and is a much more familiar concept that will make it easier to
+adopt Carbon coming from other languages. For Example when working with literals
+represented as hex, having a distinct character literal type makes this much
+simpler to work:
+
+```
+fn getSingleEAcuteHex(var combinedAcute: EBCDICChar) -> EBCDICChar {
+    var eAcute: EBCDICChar = '\u{E0}';
+
+    while (eAcute != combinedAcute) {
+        eAcute += 1;
+    }
+    return eAcute;
+}
+
+    var combinedEAcute: EBCDICChar = '\u{65}\u{300}';
+    var a: auto = getSingleEAcuteHex(combinedEAcute);
+```
+
+This allows us to use specific operations designed to manipulate the literals
+value. Whereas with a String, we would need to have additional logic to store,
+check and assign the specific value:
+
+```
+fn getSingleEAcuteHex(var combinedAcute: String) -> String {
+    var eAcute: String = "\u{E0}";
+    var accentValue: i32 = 0;
+
+    while (eAcute != combinedAcute) {
+        accentValue += 1;
+        eAcute = "\u{E{accentValue}}";
+    }
+    return eAcute;
+}
+
+    var combinedEAcute: String = "\u{65}\u{300}";
+    var a: auto = getSingleEAcuteHex(combinedEAcute);
+```
+
+Using a `String` also leads to further issues regarding familiarty with other
+languages and and leaves the logic generally open to errors, needing further
+logic to check for a valid input.
+
+Further, this design follows other standards set in place by previous proposals.
+For example following the
+[String Literals: Escaping Sequence](https://github.com/carbon-language/carbon-lang/blob/trunk/proposals/p0199.md#escape-sequences-1)
+and representing characters as integers with the behaviour inline with
+[Integer Literals](https://github.com/carbon-language/carbon-lang/blob/trunk/proposals/p0143.md).
+This also supports our goal for
+[Interoperability with and migration from existing C++ code](/docs/project/goals.md#interoperability-with-and-migration-from-existing-c-code)
+by ensuring that every kind of character literal that exists in C++ can be
+represented in a Carbon character literal. This is done in a way that is natural
+to adopt, understand, easy to read by having explicit character types mapped to
+the C++ character types and the correct associated encoding.
+
+## Alternatives considered
+
+### No Distinct Character Literal
+
+In principle a character literal can be represented by reusing string literals.
+//TODO: Add Description
+
+```
+    var b: String = "b";
+    var c: String = "c";
+    if (c > b) {
+        b = c + b;
+    }
+```
+
+We would simply this as a single element string. However it terms of
+readablility, if we had a distinct lexical syntax for character literals versus
+string literals, this would be more inline with Carbon's language design goals
+related to self documenting code, easy to read, understand, write and C++
+interopability.
```

Comment:
This is too generic a justification, here and in the later alternatives. We are looking for a rationale that specifically applies to the difference between the two alternatives.